go.mod

```diff
@@ -12,56 +12,58 @@ require (
 	github.com/elazarl/goproxy v0.0.0-20191011121108-aa519ddbe484 // indirect
 	github.com/evanphx/json-patch v4.5.0+incompatible // indirect
 	github.com/fsouza/go-dockerclient v1.6.3
-	github.com/googleapis/gnostic v0.3.1 // indirect
-	github.com/opencontainers/runc v1.0.0-rc90
-	github.com/opencontainers/runtime-spec v1.0.1
+	github.com/opencontainers/runc v1.0.0-rc91.0.20200707015106-819fcc687efb
+	github.com/opencontainers/runtime-spec v1.0.3-0.20200520003142-237cc4f519e2
 	github.com/openshift/api v0.0.0-20200326160804-ecb9283fe820
 	github.com/openshift/client-go v0.0.0-20200326155132-2a6cd50aedd0
 	github.com/openshift/imagebuilder v1.1.6
 	github.com/openshift/library-go v0.0.0-20200327125526-163b2f0d6264
 	github.com/openshift/source-to-image v1.3.0
 	github.com/pkg/errors v0.9.1
 	github.com/seccomp/containers-golang v0.0.0-20190710130535-8d1ed6a872a5 // indirect
-	github.com/spf13/cobra v0.0.7
+	github.com/spf13/cobra v1.0.0
 	github.com/spf13/pflag v1.0.5
-	golang.org/x/sys v0.0.0-20200327173247-9dae0f8f5775
-	k8s.io/api v0.18.0
-	k8s.io/apimachinery v0.18.0
-	k8s.io/client-go v0.18.0
-	k8s.io/component-base v0.18.0
-	k8s.io/klog v1.0.0
-	k8s.io/kubectl v0.18.0
-	k8s.io/kubernetes v1.18.0
+	golang.org/x/sys v0.0.0-20200622214017-ed371f2e16b4
+	k8s.io/api v0.19.0-rc.3
+	k8s.io/apimachinery v0.19.0-rc.3
+	k8s.io/client-go v0.19.0-rc.3
+	k8s.io/component-base v0.19.0-rc.3
+	k8s.io/klog/v2 v2.2.0
+	k8s.io/kubectl v0.19.0-rc.3
+	k8s.io/kubernetes v1.19.0-rc.3
 )
 
 replace (
 	github.com/containerd/containerd => github.com/containerd/containerd v1.3.2
 	github.com/docker/docker => github.com/openshift/moby-moby v1.4.2-0.20190308215630-da810a85109d
 	github.com/docker/libnetwork => github.com/docker/libnetwork v0.8.0-dev.2.0.20171107005402-dcf79f8c7fcb
 	github.com/godbus/dbus => github.com/godbus/dbus v0.0.0-20181101234600-2ff6f7ffd60f
+	// testing changes
+	github.com/opencontainers/runc => github.com/opencontainers/runc v1.0.0-rc90
+	github.com/opencontainers/runtime-spec => github.com/opencontainers/runtime-spec v1.0.1
```

Comment:
Is this replace necessary?

Reply:
I'm removing this replace block, and running `go mod tidy`. As you can see it skewed `runc` and `runtime-spec` a bit. Should we keep with a replace statement instead?

Reply:
I end up adding the replace statements again, due the errors we can see here: https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_builder/167/pull-ci-openshift-builder-master-unit/1292732924029833216/build-log.txt